### PR TITLE
[IMP][9.0] stock_batch_picking: add hook to inject custom sorting on …

### PR DESCRIPTION
**Pull request by i-vyshnevska**
…report

### CODE REVIEW

**Review by andreampiovesana — APPROVED** [review]
nice

**Comment by rousseldenis** [comment]
@i-vyshnevska Nice. I will fix travis issue

**Comment by rousseldenis** [comment]
@i-vyshnevska Could you rebase ?

**Comment by i-vyshnevska** [comment]
@rousseldenis rebased

File: stock_batch_picking/report/print_batch.py [hunk 1]
```diff
@@ -50,3 +50,8 @@ def set_context(self, objects, data, ids, report_type=None):
             new_objects.append(self._aggregation_class(batch, pack_operations))
         return super(PrintBatch, self).set_context(new_objects, data, ids,
                                                    report_type=report_type)
+
+    def get_sorted_batch_pickings(self, pickings):
```
Comment (rousseldenis) [inline stock_batch_picking/report/print_batch.py:55]: Could you put @api.model ?

**Review by rousseldenis — CHANGES_REQUESTED** [review]

**Review by rousseldenis — APPROVED** [review]

**Review by sebalix — APPROVED** [review]

**Pull request merged**